data-model/src/main/avro/eventfields/grpc/Grpc.avdl

```diff
@@ -7,6 +7,9 @@ protocol GrpcProtocol {
     union { null, string } body = null;
     int size = 0;
     map<string> metadata = {};
+    union { null, string } service = null;
```

Comment:
I dont these fields are necessary. Since url and path are populated in the http field and service in the event field itself.

Reply:
The `service` in the event field could be different from this field. 
Aren't `http` and `grpc` being populated mutually exclusively? For GRPC protocol spans, do we look at `http`? I don't think we should and hence we need `url` and `path` here too IMO.

Reply:
That means you will be moving protocol enrichment code into the spans normalizer.

Reply:
I had the same concern. For now I have removed the field of URL from GRPC message. Added the named fields for rpc metadata that we might need in trace enrichment.
Also added a record for Rpc which will have the info from the new tags defined by Otel.